src/duckula/middleware.clj

```diff
@@ -12,7 +12,8 @@
   [handler-fn]
   (-> handler-fn
       (ring-json/wrap-json-body {:keywords? true})
-      (ring-defaults/wrap-defaults ring-defaults/api-defaults)))
+      (ring-defaults/wrap-defaults ring-defaults/api-defaults)
+      (ring-json/wrap-json-response)))
```

Comment:
This is a breaking change! Our internal middleware stack will have to make similar change - otherwise we will get Ring errors. 
That said, this makes the whole setup much cleaner and makes Duckula behave more like a Ring middleware because we do not force JSON as output format.